app/javascript/App.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import Greeting from './Greeting';
+
+const App = () => {
+    return (
+        <div>
+            <Greeting />
+        </div>
+    );
```

Comment:
- [ ] I did not see any greetings(messages) on the screen even after creating 5 messages. Kindly fix the issue.

![one-app](https://user-images.githubusercontent.com/53124548/213677899-34be94bb-1443-4c87-94e6-e2be338de361.png)